presto-tests/src/test/java/io/prestosql/tests/TestTpchDistributedQueries.java

```diff
@@ -116,4 +118,23 @@ public void testTableSampleSystem()
         }
         assertTrue(sampleSizeFound, "Table sample returned unexpected number of rows");
     }
+
+    public void testShowTables()
+    {
+        Session session = testSessionBuilder()
```

Comment:
- inline (it's used only once)
- create a helper method that constructs a Session for given schema